### ISSUE TITLE
Phet-io mirror wrapper doesn't work and studio doesn't launch

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/capacitor-lab-basics#291]
For https://github.com/phetsims/QA/issues/642. Seen on Win 10 Chrome. Found in https://phet-dev.colorado.edu/html/capacitor-lab-basics/1.6.20-rc.1/phet-io/wrappers/mirror-inputs/mirror-inputs.html and https://phet-dev.colorado.edu/html/capacitor-lab-basics/1.6.20-rc.1/phet-io/wrappers/instance-proxies/instance-proxies.html?sim=capacitor-lab-basics&relativeSimPath.
The mirror image wrapper does not show a sim either upstream or downstream. In the studio (instance proxies) changes can be made, but pressing launch leads to a blank screen.

![launchblank](https://user-images.githubusercontent.com/41024075/117355703-25485680-ae70-11eb-8db0-a167e17e7d47.gif)
<img width="905" alt="nomirror" src="https://user-images.githubusercontent.com/41024075/117355706-25e0ed00-ae70-11eb-92b4-0132c2e5c8bc.png">

**Comment by KatieWoe** [comment]
@jonathanolson have you ever run into this trouble before while MRing this version?

**Comment by KatieWoe** [comment]
This is happening with resistance in a wire as well. https://phet-dev.colorado.edu/html/resistance-in-a-wire/1.3.13-phetiorc.1/resistance-in-a-wire_en-phetio.html?phetioStandalone

**Comment by zepumph** [comment]
This issue seems to be because older phet-io releases make assumptions that it is deployed into the version directory, and not nested in the phet-io/ directory. I can't be sure though.

**Comment by jonathanolson** [comment]
Got this fixed up above, links should be working for next RC testing (don't recommend testing yet): https://phet-dev.colorado.edu/html/capacitor-lab-basics/1.6.20-rc.2/phet-io/wrappers/mirror-inputs/mirror-inputs.html and https://phet-dev.colorado.edu/html/resistance-in-a-wire/1.3.13-phetiorc.2/wrappers/mirror-inputs/mirror-inputs.html

**Comment by jonathanolson** [comment]
Reopening for tracking.

**Comment by phet-steele** [comment]
@jonathanolson every single instance-proxies wrapper we are asked to test in phetsims/QA/issues/649 fails to "Launch" on my Win 10 Chrome. They each fail with one of two errors.

The following sims fail with this error:
`Uncaught ReferenceError: QueryStringMachine is not defined`

- https://phet-dev.colorado.edu/html/balloons-and-static-electricity/1.3.17-phetiorc.1/wrappers/instance-proxies/instance-proxies.html?sim=balloons-and-static-electricity&relativeSimPath
- https://phet-dev.colorado.edu/html/beers-law-lab/1.6.16-phetiorc.1/wrappers/instance-proxies/instance-proxies.html?sim=beers-law-lab&launchLocalVersion
- https://phet-dev.colorado.edu/html/build-an-atom/1.5.19-phetiorc.1/wrappers/instance-proxies/instance-proxies.html?sim=build-an-atom&relativeSimPath
- https://phet-dev.colorado.edu/html/charges-and-fields/1.0.21-phetiorc.1/wrappers/instance-proxies/instance-proxies.html?sim=charges-and-fields&launchLocalVersion
- https://phet-dev.colorado.edu/html/color-vision/1.2.17-phetiorc.1/wrappers/instance-proxies/instance-proxies.html?sim=color-vision&launchLocalVersion
- https://phet-dev.colorado.edu/html/concentration/1.5.15-phetiorc.1/wrappers/instance-proxies/instance-proxies.html?sim=concentration&launchLocalVersion
- https://phet-dev.colorado.edu/html/faradays-law/1.3.18-phetiorc.1/wrappers/instance-proxies/instance-proxies.html?sim=faradays-law&launchLocalVersion
- https://phet-dev.colorado.edu/html/forces-and-motion-basics/2.1.107-phetiorc.1/wrappers/instance-proxies/instance-proxies.html?sim=forces-and-motion-basics&launchLocalVersion
- https://phet-dev.colorado.edu/html/john-travoltage/1.4.17-phetiorc.1/wrappers/instance-proxies/instance-proxies.html?sim=john-travoltage&relativeSimPath
- https://phet-dev.colorado.edu/html/molecules-and-light/1.3.14-phetiorc.1/wrappers/instance-proxies/instance-proxies.html?sim=molecules-and-light&launchLocalVersion

Then these sims fail with this error:
`ReferenceError: SimIFrameClient is not defined`

- https://phet-dev.colorado.edu/html/capacitor-lab-basics/1.6.20-rc.3/phet-io/wrappers/instance-proxies/instance-proxies.html?sim=capacitor-lab-basics&relativeSimPath
- https://phet-dev.colorado.edu/html/circuit-construction-kit-black-box-study/1.1.17-phetiorc.1/wrappers/instance-proxies/instance-proxies.html?sim=circuit-construction-kit-black-box-study&relativeSimPath
- https://phet-dev.colorado.edu/html/energy-skate-park-basics/1.3.15-phetiorc.1/wrappers/instance-proxies/instance-proxies.html?sim=energy-skate-park-basics&relativeSimPath
- https://phet-dev.colorado.edu/html/resistance-in-a-wire/1.3.13-phetiorc.3/wrappers/instance-proxies/instance-proxies.html?sim=resistance-in-a-wire&relativeSimPath

**Comment by KatieWoe** [comment]
It looks like some mirror inputs may fail as well. This is faradays law.
<img width="960" alt="mirrorfaraday" src="https://user-images.githubusercontent.com/41024075/118875030-d0153780-b8a8-11eb-8745-9a4a5ee0c6b1.png">

**Comment by KatieWoe** [comment]
Other sims like balloons and static electricity have the mirror wrapper issue still too.
<img width="770" alt="mirrorballoon" src="https://user-images.githubusercontent.com/41024075/118894037-b384f980-b8c0-11eb-9473-de06174917a2.png">

**Comment by jonathanolson** [comment]
@kathy-phet, it seems like a lot of these have been broken for a while, and aren't broken by this maintenance release. Should this be looked into separately, or should I be fixing it as part of this release?

**Comment by kathy-phet** [comment]
@KatieWoe - Are all these failures exhibited on the current published version too? See this list: 
https://docs.google.com/spreadsheets/d/e/2PACX-1vReSXUUxC63CQCKHIoO6qY4ycWrqgw8eaX25io3_r2hRmO3RbXuyNRRMz5J2C1c5mCN9nxumvrv6snj/pubhtml?gid=2141068918&single=true

**Comment by kathy-phet** [comment]
Another option would be to deploy these but then manually delete the phet-io versions from the production servers. That way the clients would be server the existing working phet-io versions, and we can still use the automated deploy tools for phet maintenance? I think.

**Comment by KatieWoe** [comment]
Beers Law Lab had this issue when trying to launch from studio:
<img width="413" alt="beerslawstudio" src="https://user-images.githubusercontent.com/41024075/119905833-3124b700-bf0a-11eb-81da-a1bfb1cebdaa.png">
Otherwise, studio and mirror wrapper seem to load properly in the sims linked in https://github.com/phetsims/capacitor-lab-basics/issues/291#issuecomment-849205773

**Comment by samreid** [comment]
If I recall correctly, we discussed this further on Thursday.  @jonathanolson do you recall that discussion and can you summarize?

**Comment by kathy-phet** [comment]
@samreid - I believe we found that these work when published, but do not work as dev or rc versions. So we decided to proceed with the maintenance release and then check that they are indeed working on the published website.

**Comment by liam-mulhall** [comment]
Studio still isn't launching in https://phet-dev.colorado.edu/html/resistance-in-a-wire/1.3.13-phetiorc.4/wrappers/instance-proxies/instance-proxies.html?sim=resistance-in-a-wire&relativeSimPath for https://github.com/phetsims/QA/issues/653.

**Comment by jonathanolson** [comment]
Can we check these on the production site now @KatieWoe?

**Comment by KatieWoe** [comment]
I must have missed this when it was first assigned to me. I apologize. According to @kathy-phet it looks like launching is still broken in published. Discussion also seen in https://github.com/phetsims/qa/issues/859. What can I do to help?

**Comment by KatieWoe** [comment]
I'm also unsure where to find a list of the published Instance Proxies for future testing/work. Can someone provide this?

**Comment by arouinfar** [comment]
Capacitor Lab: Basics has only partial PhET-iO. It was never designed or released publicly. Anything that would have been shared would have been with a research partner with the understanding it was a one-off with no real support.
